### ISSUE TITLE
[corsarored] Connection refused: Connection refused (Config)

### ISSUE THREAD

**Issue by 100ferhas** [Jackett/Jackett#9454]
I have Jackett plugin (docker) installed on my unraid server, it's working fine for all indexers but _Corsaro.red_, I'm unable to add it because is throwing following error:

<img width="882" alt="Schermata 2020-09-05 alle 11 36 50" src="https://user-images.githubusercontent.com/46828393/92303292-c09ccb00-ef73-11ea-9590-284fc5532134.png">

This is the logged error:
```
09-05 11:25:00 Error On attempt 1 checking for results from Corsaro.red: Connection refused

09-05 11:25:01 Error On attempt 2 checking for results from Corsaro.red: Connection refused

09-05 11:25:01 Error On attempt 3 checking for results from Corsaro.red: Connection refused

09-05 11:25:02 Error System.Net.Http.HttpRequestException: Connection refused

---> System.Net.Sockets.SocketException (111): Connection refused
at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
--- End of inner exception stack trace ---
at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookiesAndRetry(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 469
at Jackett.Common.Indexers.CorsaroRed.SendApiRequestLatest() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CorsaroRed.cs:line 139
at Jackett.Common.Indexers.CorsaroRed.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CorsaroRed.cs:line 170
at Jackett.Common.Indexers.CorsaroRed.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CorsaroRed.cs:line 102
at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)

at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 70 System.Net.Http.HttpRequestException: Connection refused
---> System.Net.Sockets.SocketException (111): Connection refused
at System.Net.Http.ConnectHelper.ConnectAsync(String host, Int32 port, CancellationToken cancellationToken)
--- End of inner exception stack trace ---
at Jackett.Common.Indexers.BaseWebIndexer.RequestStringWithCookiesAndRetry(String url, String cookieOverride, String referer, Dictionary`2 headers) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/BaseIndexer.cs:line 469
at Jackett.Common.Indexers.CorsaroRed.SendApiRequestLatest() in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CorsaroRed.cs:line 139
at Jackett.Common.Indexers.CorsaroRed.PerformQuery(TorznabQuery query) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CorsaroRed.cs:line 170
at Jackett.Common.Indexers.CorsaroRed.ApplyConfiguration(JToken configJson) in /home/vsts/work/1/s/src/Jackett.Common/Indexers/CorsaroRed.cs:line 102
at Jackett.Server.Controllers.IndexerApiController.UpdateConfig(ConfigItem[] config) in /home/vsts/work/1/s/src/Jackett.Server/Controllers/IndexerApiController.cs:line 94
at Microsoft.AspNetCore.Mvc.Infrastructure.ActionMethodExecutor.TaskOfIActionResultExecutor.Execute(IActionResultTypeMapper mapper, ObjectMethodExecutor executor, Object controller, Object[] arguments)

at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeActionMethodAsync>g__Awaited|12_0(ControllerActionInvoker invoker, ValueTask`1 actionResultValueTask)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeNextActionFilterAsync>g__Awaited|10_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Rethrow(ActionExecutedContextSealed context)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.Next(State& next, Scope& scope, Object& state, Boolean& isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ControllerActionInvoker.<InvokeInnerFilterAsync>g__Awaited|13_0(ControllerActionInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeFilterPipelineAsync>g__Awaited|19_0(ResourceInvoker invoker, Task lastTask, State next, Scope scope, Object state, Boolean isCompleted)
at Microsoft.AspNetCore.Mvc.Infrastructure.ResourceInvoker.<InvokeAsync>g__Awaited|17_0(ResourceInvoker invoker, Task task, IDisposable scope)
at Microsoft.AspNetCore.Routing.EndpointMiddleware.<Invoke>g__AwaitRequestTask|6_0(Endpoint endpoint, Task requestTask, ILogger logger)
at Microsoft.AspNetCore.Authentication.AuthenticationMiddleware.Invoke(HttpContext context)
at Jackett.Server.Middleware.CustomExceptionHandler.Invoke(HttpContext httpContext) in /home/vsts/work/1/s/src/Jackett.Server/Middleware/CustomExceptionHandler.cs:line 70
```

**Comment by 100ferhas** [comment]
#9428 is an empty issue

**Comment by ilike2burnthing** [comment]
Are you using a VPN or proxy?

Can you access the site in browser from the same IP?

**Comment by 100ferhas** [comment]
yes i can reach it from browser without problems, i am not using any vpn or proxy.

**Comment by ilike2burnthing** [comment]
Have you tried restarting Jackett?

**Comment by 100ferhas** [comment]
I ve restarted jackett container and the whole system several times, but it still gives connection refused.

**Comment by ilike2burnthing** [comment]
Ah! Sorry, didn't mean to close the thread.

@garfield69 any ideas?

**Comment by ilike2burnthing** [comment]
Have you run tests and searches for your other indexers without issues? Just I noticed you have KAT there with a green tick, and it's currently offline, so it shouldn't work.

**Comment by garfield69** [comment]
No clue. The corsarored indexer is working for me so this suggests its an issue at your end.
Whatever the issue is has nothing to do with the Jackett app.

Recently saw a ticket where a docker user was having failures with his Jackett YTS indexer and in the end he factory reset docker after which his Jackett YTS indexer started working again #9452

**Comment by 100ferhas** [comment]
> Have you run tests and searches for your other indexers without issues? Just I noticed you have KAT there with a green tick, and it's currently offline, so it shouldn't work.

I've just re-run tests and actually KAT is not reachable, I've no problems with searches.


> No clue. The corsarored indexer is working for me so this suggests its an issue at your end.
> Whatever the issue is has nothing to do with the Jackett app.
> 
> Recently saw a ticket where a docker user was having failures with his Jackett YTS indexer and in the end he factory reset docker after which his Jackett YTS indexer started working again #9452

I tried to delete and rebuild the docker image I was using and I always have the same problem with Corsaro.red, I also tried to use a different docker image but with the same result. I can't add Corsaro.red to my indexers.

**Comment by garfield69** [comment]
I believe that #9452 **docker reset** was not about the docker jackett image, but the docker app itself.

**Comment by 100ferhas** [comment]
I've found what is the cause of the error. I don't know why my ISP DNS is resolving corsaro.red as localhost:

![image](https://user-images.githubusercontent.com/46828393/92314945-1490c980-efdf-11ea-8d6f-89cc1a61e35e.png)

On my other pc (where i was able to reach correctly the website), I was using Google DNS that resolves correctly the corsaro.red domain. I've now changed DNS also on my server and now everything is working fine.

Thanks for support @garfield69 @ilike2burnthing !
